test/tests_for_sdkmr/ee2_SDKMethodRunner_test.py

```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf-8 -*-
+test / tests_for_sdkmr / ee2_SDKMethodRunner_test.pytest / tests_for_sdkmr / ee2_SDKMethodRunner_test.py  # -*- coding: utf-8 -*-
```

Comment:
This looks wrong...?

Reply:
Thanks, not sure how this got in there, i wonder what I am misclicking? Maybe I'm dragging something.

Reply:
:+1: